src/frontend/components/UploadManager/index.tsx

```diff
@@ -76,7 +78,11 @@ export const UploadManager = ({ children }: React.PropsWithChildren<{}>) => {
             key,
             presignedPost.fields[key],
           ]),
-          ...(([modelName.VIDEOS, modelName.THUMBNAILS].includes(objectType)
+          ...(([
```

Comment:
Shouldn't you update this component tests ?

Reply:
All tests performed on UploadManager are done with the same video object. Behavior of the component depending on the object type is not tested.